pkg/customAnalyzer/docker/docker.go

```diff
@@ -0,0 +1,136 @@
+package docker
```

Comment:
I am reluctant to take a hard dependency like docker :(  We need to discuss also if this is the approach we want to take